extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -341,9 +341,43 @@ export class AmpAutocomplete extends AMP.BaseElement {
     this.container_ = this.createContainer_();
     this.element.appendChild(this.container_);
 
+    this.initializeListeners_();
+
     return Promise.resolve();
   }
 
+  /**
+   * Initializes listeners for keydown, mousedown, focus, etc. events.
+   * @private
+   */
+  initializeListeners_() {
+    this.inputElement_.addEventListener(
+      'touchstart',
+      () => {
```

Comment:
Nit: Does it matter what the return type is?

`() => this.checkFirstInteractionAndMaybeFetchData_()`

Reply:
Thanks for your comments. AFAIK `addEventListener` callbacks should not return anything. That's why I left code unchanged to avoid any implicit returns from methods like `checkFirstInteractionAndMaybeFetchData_` (which returns a promise).